core/src/main/java/org/hisp/dhis/android/core/configuration/ConfigurationStoreImpl.java

```diff
@@ -35,11 +35,12 @@
 import android.support.annotation.Nullable;
 
 import org.hisp.dhis.android.core.data.database.DatabaseAdapter;
+import org.hisp.dhis.android.core.common.DeletableStore;
 
 import static org.hisp.dhis.android.core.utils.StoreUtils.sqLiteBind;
 import static org.hisp.dhis.android.core.utils.Utils.isNull;
 
-public class ConfigurationStoreImpl implements ConfigurationStore {
+public class ConfigurationStoreImpl implements ConfigurationStore, DeletableStore {
```

Comment:
Store interfaces should extend from DeletableStore, for avoid duplicate code between interfaces because there are stores as OrganisationUnitStore with delete method also